### PR TITLE
Update sl-jdbc.md

### PR DESCRIPTION
**Pull request by rpourzand**
- update to filter list for WHERE
- updates for the ability to get the agg_time_dimension name for measures
- updates for the ability to get queryable granularities for more than just a primary time dim

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]: @rpourzand , what does predicate mean in this sentence?

```suggestion
In the API, you can use where filters either as a filter list or string. For production applications, it's better to use a filter list. This format helps you take advantage of all the benefits from the predicate pushdown where possible.
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
Where Filters have a few objects that you can use:
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
Use the following example to query using a `where` filter with the string format:
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
Note: If you prefer a more strongly typed `where` clause, you can optionally use the `TimeDimension` feature. This helps separate out categorical dimensions from time-related ones. The `TimeDimesion` input takes the time dimension name and also requires granularity, like this: `TimeDimension('metric_time', 'MONTH')`.
```

**Comment by mirnawong1** [comment]
lgtm!

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md
Comment (rpourzand) [reply website/docs/docs/dbt-cloud-apis/sl-jdbc.md]: So a predicate is just an expression to determine what rows in a database apply to a particular query. For example, if I filter in a WHERE clause on a particular dimension value, the database is basically going to search to determine what values in the DB apply to this query. If this can be pushed down to the database (predicate pushdown)), it's much faster.

Comment (rpourzand) [reply website/docs/docs/dbt-cloud-apis/sl-jdbc.md]: We could link out some generic docs on predicate pushdown.

Comment (mirnawong1) [reply website/docs/docs/dbt-cloud-apis/sl-jdbc.md]: Predicate Pushdown points to the where or filter clause which affects the number of rows returned. It basically relates to which rows will be filtered, not which columns. For this reason, while applying the filter on a nested column as ‘library.books’ to merely return records with the values that are not null, the predicate pushdown function will make parquet read blocks that contain no null values for the specified column.

**Review by marcodamore — APPROVED** [review]

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
Use this query to fetch queryable granularities for a list of metrics. This API request allows you to only show the time granularities that make sense for the primary time dimension of the metrics (such as `metric_time`), but if you want queryable granularities for other time dimensions, you can use the `dimensions()` call, and find the column queryable_granularities.
```

**Comment by mirnawong1** [comment]
waiting until roxi is back from ooo

**Comment by rpourzand** [comment]
hey @mirnawong1 I edited this one to remove some things that are still in flight, so pending your final review it can be shipped

**Review by mirnawong1 — APPROVED** [review]
lgtm @rpourzand ! happy to merge this for you if you're ok with it! ✨

**Pull request merged**
